pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -686,6 +686,9 @@ func allocatableResources(memory, cpu string) v1.ResourceList {
 
 // DaemonSets should not place onto nodes with insufficient free resource
 func TestInsufficientCapacityNodeDaemonDoesNotLaunchPod(t *testing.T) {
+	if utilfeature.DefaultFeatureGate.Enabled(features.ScheduleDaemonSetPods) {
```

Comment:
Prefer to set `ScheduleDaemonSetPods` to `false` for those test until we upgrade to GA.

Reply:
Klaus, I have disabled and enabled the featuregate as you suggested. PTAL.